test/twine_test.coffee

```diff
@@ -164,6 +164,28 @@ suite "Twine", ->
       node = setupView(testView, key: "<script>")
       assert.equal node.innerHTML, "&lt;script&gt;"
 
+    test "should be the first binding to run on change event", ->
+      testView = "<input type=\"text\" bind-event-change=\"eventFunc()\" data-bind=\"val\">"
+      node = setupView(testView, context = {
+        val: 1,
+        eventFunc: () -> this.val = this.val * 2
+      })
+      node.value = 2
+      triggerEvent node, "change"
+
+      assert.equal context.val, 4
```

Comment:
This test fails when the `type == 'bind'` --> `unshift` logic (`twine.coffee:83 - 85`) is removed from the `bind` function.
```coffeescript
      if type == 'bind'
        bindingConstructors.unshift({constructor, definition})
      else
        bindingConstructors.push({constructor, definition})
```
gives: 
![image](https://cloud.githubusercontent.com/assets/2578036/16645323/193fadaa-43f1-11e6-98fc-2f21a5aebf48.png)

while just

```coffeescript
      bindingConstructors.push({constructor, definition})
```

gives:
![image](https://cloud.githubusercontent.com/assets/2578036/16645318/111bf5b6-43f1-11e6-8086-68a29c5dd117.png)